requirements.txt

```diff
@@ -0,0 +1,6 @@
+Cython==0.25.2
+confluent-kafka==0.11.0
+fastavro==0.16.3
+git+https://github.com/confluentinc/confluent-kafka-python#egg=confluent-kafka[avro]
```

Comment:
We don't need these two dependencies since they are already dependent from eventsourcing_helpers

Reply:
I see no issue in being explicit x2 :-D

Reply:
I think it looks fine. If eventsourcing_helpers is not gonna change dependencies much I think being explicit on its dependencies isn't a big deal personally.